lib/modules/dosomething/dosomething_user/dosomething_user.module

```diff
@@ -106,8 +106,7 @@ function dosomething_user_menu() {
   $items['northstar/%'] = array(
     'page callback' => 'dosomething_user_get_view',
     'page arguments' => array(1),
-    'access callback' => 'user_view_access',
-    'access arguments' => array(1),
+    'access callback' => TRUE,
```

Comment:
Sorry was at 1:1 w/ matt -- Yeah we always want this to be true, cause anonymous users shouldn't be blocked by it. I don't see any danger in this either, northstar id's are public